src/eval.rs

```diff
@@ -27,6 +27,8 @@ pub struct MiriConfig {
     pub args: Vec<String>,
     /// The seed to use when non-determinism or randomness are required (e.g. ptr-to-int cast, `getrandom()`).
     pub seed: Option<u64>,
+    /// The stacked borrow id to report about
+    pub tracked_id: Option<PtrId>,
```

Comment:
We never call these "borrow ID". The usual name is "Pointer tag". Yes, tags include all `PtrId` and `\bottom`, aka `Option<PtrId>`, but still -- the term "pointer ID" is entirely an implementation detail of Stacked Borrows and should not be used outside.

I propose `tracked_pointer_tag`.